src/link/main.c

```diff
@@ -199,6 +203,9 @@ int main(int argc, char *argv[])
 		case 'w':
 			isWRA0Mode = true;
 			break;
+		case 'x':
```

Comment:
I'd probably make no padding imply tiny mode, instead of enforcing it with a check.

Reply:
Alternate implementation (would be more useful, but more work): no padding + no tiny mode works like tiny mode, but with an unbounded ROM0 size.

After all, $8000 is an irrelevant mark when the output is not a ROM.

Reply:
Considering that this was to be used as a temporary file to INCBIN into a ROM, I'd say that tiny mode would be very reasonable. I'll go ahead and make it imply tiny mode and remove the check.

Reply:
Implies tiny mode now, if you haven't seen it.

Reply:
I did; I can't close the conversation or anything, though.